test/travis_ci.sh

```diff
@@ -70,7 +70,35 @@ function set_root {
 
 function setup_env {
     sudo apt-get update
-    sudo apt-get --assume-yes install python-pip zlib1g-dev python-lxml libxml2-dev libxslt1-dev python-dev libboost-dev libboost-python-dev libssh-dev libcurl4-openssl-dev libtool
+    sudo apt-get --assume-yes install python-pip zlib1g-dev python-lxml libxml2-dev libxslt1-dev python-dev libboost-dev libboost-python-dev libcurl4-openssl-dev libtool
+
+    cd ~
+    git clone https://github.com/Kitware/CMake.git
```

Comment:
It is probably safer to do a 'git checkout <commit-id>' of a particular commit after doing the git clone. This way, we will always use the version we have tested with.